docs/Antora/modules/getting_started/pages/quick_start_application.adoc

```diff
@@ -621,44 +432,17 @@ Cpp::
 --
 [source,cpp]
 ----
-#include <opendaq/opendaq.h>
-#include <iostream>
-#include <chrono>
-#include <thread>
-
 int main(int argc, const char* argv[])
 {
-    ...
+    // ...
 
-    // Get the first device channel
+    // Get the first signal of the first device's channel
```

Comment:
The first letters of the words "signal" and "device" should perhaps be capitalized throughout the docs, so as to keep the style uniform

Reply:
I think I might keep that for a separate PR, as we're somewhat inconsistent on this throughout the whole documentation. I'll maybe go and actually change everything to lowercase, it's hard to define what should be capitalized and what not (currently it's a bit arbitrary).